hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -432,10 +432,15 @@ def _get_dates(self):
         return dates
 
     def _get_line_name(self, project, task=None):
-        name = '{}'.format(project.name)
         if task:
-            name += ' - {}'.format(task.name)
-        return name
+            return _('%(project)s - %(task)s') % {
```

Comment:
This shouldn't be translatable. You are translating `%(project)s - %(task)s`, not the name of the project and the name of the task. That's done automatically by ORM.

Reply:
That's what I'm after - LTR/RTL support

Reply:
You don't get it this way. That's done automatically by Odoo AFAIK, flipping the order. Any way, this is always a bad idea to pass to translate these things, as a bad translation breaks the system: imagine someone translates this as `%(proyecto)s - %(tarea)s` in Spanish...

Reply:
Yeah, that would be a disaster. Yet I'm not sure that Odoo does the auto-reverse automatically since it's just not correct way to handle RTL/LTR. `%s - %s` also won't work for that :)

Reply:
But do you RTL specific cases? If not, don't worry about these weird cases.

Reply:
Well, don't know, that's just common practice (e.g. in UI-heavy platforms) to write positional formatting strings as translatable, why not to do the same in Odoo?

Reply:
I would prefer to not put as translatable, and use another technique for determining the order (maybe there's an RTL variable or something?). Any way, the translator is not going to know these internals for doing this right.

Reply:
Unless the localization is done by integrator. I don't think there will be strong pros/cons point from either side

Reply:
This is a no-go for me. I can assure OCA (and Odoo) translations are not done with these skills, so better to leave it fixed.